patches/chrome-browser-ui-views-translate-translate_bubble_view.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/views/translate/translate_bubble_view.cc b/chrome/browser/ui/views/translate/translate_bubble_view.cc
+index c67c22c3e4cb8555a2db49c110839b5be9edfc2b..4b0edc53b026538addd2ba66db68cfefe8bff6fc 100644
+--- a/chrome/browser/ui/views/translate/translate_bubble_view.cc
++++ b/chrome/browser/ui/views/translate/translate_bubble_view.cc
+@@ -128,7 +128,7 @@ views::Widget* TranslateBubbleView::ShowBubble(
+           source_language, target_language));
+   std::unique_ptr<TranslateBubbleModel> model(
+       new TranslateBubbleModelImpl(step, std::move(ui_delegate)));
+-  TranslateBubbleView* view = new TranslateBubbleView(
++  TranslateBubbleView* view = new BraveTranslateBubbleView(
```

Comment:
rather than change this line, I think it's better to add BRAVE_SHOW_BUBBLE_ after it. Then if ENABLE_BRAVE_TRANSLATE is `false` `BRAVE_SHOW_BUBBLE_` does nothing. If it's true then it replaces `view` with BraveTranslateBubbleView. This also eliminates all the guards inside `BraveTranslateBubbleView` and any issues with android. brave_translate_bubble_view.h/.cc should also go inside enable_brave_translate guards in gn

Reply:
Could we keep this patch line but replace BraveTranslateBubbleView back to TranslateBubbleView in the chromium_src override instead if `ENABLE_BRAVE_TRANSLATE_EXTENSION` is false? 
This would also eliminates the guards inside `BraveTranslateBubbleView and guard brave_translate_bubble_view.h/.cc inside gn`.
Currently did this in https://github.com/brave/brave-core/commit/6acee43d2164e814123b3bf70b11dbd469c32a0d, wdyt?

Reply:
Updated as suggested in https://github.com/brave/brave-core/pull/3103/commits/60e70deef4adfd535e25e0b481afd2683f52e604